doc/exts/pylint_messages.py

```diff
@@ -230,7 +230,9 @@ def _write_single_message_page(category_dir: Path, message: MessageData) -> None
             stream.write(
                 f"""
 .. note::
-  This message is emitted by an optional checker which requires the ``{message.checker_module_name}`` plugin to be loaded. See: :ref:`{message.checker_module_name}`.
+  This message is emitted by :ref:`the optional '{message.checker}' checker
+  <{message.checker_module_name}>` which requires the ``{message.checker_module_name}``
+  plugin to be loaded.
```

Comment:
Hmm this is an unrelated change, I don't remember if I opened a PR for this. Maybe slightly related.